tests/block_tools.py

```diff
@@ -1394,8 +1433,8 @@ def get_challenges(
     return cc_challenge, rc_challenge
 
 
-def get_plot_dir() -> Path:
-    cache_path = DEFAULT_ROOT_PATH.parent.joinpath("test-plots")
+def get_plot_dir(plot_dir_name: str = "test-plots") -> Path:
```

Comment:
We are hardcoding "test-plots" in multiple places, so I recommend not having a default value here